test/sql/copy/csv/glob/read_csv_glob_crawl.test

```diff
@@ -0,0 +1,198 @@
+
+query I
+SELECT count(*) FROM read_csv('test/sql/copy/csv/data/glob/crawl/d/**/*.csv', auto_detect=1);
+----
+148
+
+query I
+SELECT count(*) FROM glob('test/sql/copy/csv/data/glob/crawl/d/**');
+----
+148
+
+query I
+SELECT sum(column0) FROM read_csv('test/sql/copy/csv/data/glob/crawl/d/**', auto_detect=1);
+----
+6216
+
+query I
+SELECT count(*) FROM glob('test/sql/copy/csv/data/glob/crawl/d/**/');
+----
+148
+
+query I
+SELECT count(*) FROM glob('test/sql/copy/csv/data/glob/crawl/d/**/mid/*.csv');
+----
+27
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/mid/*.csv';
+----
+27
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/mid/*/*.csv';
+----
+81
+
+query I
+SELECT count(*) FROM glob('test/sql/copy/csv/data/glob/crawl/d/**/mid/*/');
+----
+81
+
+statement error
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/mid/**/*.csv';
+----
+IO Error: Cannot use multiple '**' in one path
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/???/*/*.csv';
+----
+144
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/???/**/*.csv';
+----
+144
+
+statement error
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/mid/**/*.csv';
+----
+IO Error: No files found that match the pattern 
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/*/*/mid/**/*.csv';
+----
+108
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/???/*.csv';
+----
+147
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/???/*.csv';
+----
+9
+
+statement error
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/*/**/*.csv';
+----
+IO Error: Cannot use multiple '**' in one path
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/d2?/*/*.csv';
+----
+27
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/*/d2?/**/*.csv';
+----
+135
+
+query I
+SELECT sum(column0) FROM read_csv('test/sql/copy/csv/data/glob/crawl/d/*/*/d2?/**', auto_detect=1);
+----
+5670
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/d?0/*.csv';
+----
+40
+
+query I
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/*/**/d?0/*.csv';
+----
+39
+
+statement error
+SELECT count(*) FROM 'test/sql/copy/csv/data/glob/crawl/d/**/**/**/**/*.csv';
+----
+IO Error: Cannot use multiple '**' in one path
+
+# Test with hidden files. By default python does not return hidden files, but duckdb does.
+query II
+FROM read_csv_auto('test/sql/copy/csv/data/glob/crawl/hidden/**');
+----
+42	42
+42	42
+42	42
+
+# tests without previous directories
+query I
+select count(*) > 0 from glob('**');
+----
+true
+
+query I
+select count(*) from glob('**/samename/*');
+----
+8
+
+require notwindows
```

Comment:
I think this require can be removed here? Shouldn't this test also pass on windows now that we decided to not follow symlinks?